Gems/AWSGameLift/Code/AWSGameLiftClient/Include/Request/AWSGameLiftStartMatchmakingRequest.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzFramework/Session/ISessionRequests.h>
+
+namespace AWSGameLift
+{
+    //! AWSGameLiftPlayerInformation
+    //! Information on each player to be matched
+    //! This information must include a player ID, and may contain player attributes and latency data to be used in the matchmaking process
+    //! After a successful match, Player objects contain the name of the team the player is assigned to
+    struct AWSGameLiftPlayerInformation
+    {
+        AZ_RTTI(AWSGameLiftPlayerInformation, "{B62C118E-C55D-4903-8ECB-E58E8CA613C4}");
+        static void Reflect(AZ::ReflectContext* context);
+
+        AWSGameLiftPlayerInformation() = default;
+        virtual ~AWSGameLiftPlayerInformation() = default;
+
+        // A set of values, expressed in milliseconds, that indicates
+        // the amount of latency that a player experiences when connected to AWS Regions
+        AZStd::unordered_map<AZStd::string, int> m_latencyInMs;
+        // A collection of key:value pairs containing player information for use in matchmaking
+        // Player attribute keys must match the playerAttributes used in a matchmaking rule set
+        // Example: {"skill": "{\"N\": \"23\"}", "gameMode": "{\"S\": \"deathmatch\"}"}
+        AZStd::unordered_map<AZStd::string, AZStd::string> m_playerAttributes;
```

Comment:
So matchmaking request takes json format?

Reply:
GameLift defines its own data structure we need to convert to before sending the request.